gulp/lint.js

```diff
@@ -13,7 +14,12 @@ gulp.task('lint', ['lint:sass', 'lint:javascript']);
 
 gulp.task('lint:sass', function() {
   return gulp.src(PATHS)
-    .pipe(scssLint());
+    .pipe(plumber())
```

Comment:
I think it is. It is not "compilation errors", but only "formating warnings". It shouldn't interrupt the compilation.